lua/lean/infoview/components.lua

```diff
@@ -144,6 +144,15 @@ local function code_with_infos(t, sess)
       end
     end
 
+    --FIXME this is essentially a thunked boolean field, adding it as an event
+    --is not exactly, but I'm not sure where to stick it at the moment
+    ---@param kind GoToKind
+    local has_go_to = function(_, kind)
+      local links, err = sess:getGoToLocation(kind, info_with_ctx)
```

Comment:
This will be very slow if you call it sequentially on all subterms in the infoview.  I'd just add a hint everywhere (no matter whether `getGoToLocation` succeeds or not).

Reply:
Yes, this is a temporary hack until we add some kind of lightweight indicator of which elements have go-to defintition/declaration/etc. on the server side. hop.nvim orders hints by reachability on your keyboard, so from an ergonomic perspective I don't think I can settle for less lol. Even this hacked solution has been very fast for me though? If you're noticing performance issues we could of course make it configurable, but even so personally I'd rather wait a bit longer for the hints than have to press an additional/harder-to-reach key.